cmake/LibeventConfig.cmake.in

```diff
@@ -1,18 +1,178 @@
 # - Config file for the Libevent package
 # It defines the following variables
-#  LIBEVENT_INCLUDE_DIRS     - include directories
+#  LIBEVENT_FOUND            - true if libevent and all required components found on the system
+#  LIBEVENT_xxx_FOUND        - true if component xxx(see available components) found on the system
+#  LIBEVENT_VERSION          - libevent version in format Major.Minor.Patch
+#  LIBEVENT_INCLUDE_DIRS     - directories where libevent header is located.
+#  LIBEVENT_INCLUDE_DIR      - same as DIRS
+#  LIBEVENT_LIBRARIES        - libevent library to link against.
+#  LIBEVENT_LIBRARY          - same as LIBRARIES
+#
+# These variables are deprecated, only reserved for compatibility.
 #  LIBEVENT_STATIC_LIBRARIES - libraries to link against (archive/static)
 #  LIBEVENT_SHARED_LIBRARIES - libraries to link against (shared)
+#
+# When you try to locate the libevent libraries, you should specify which components you want to use.
+# The following table lists all available components. If none is given, all imported targets will used.
+#  event_core        - the core functons of libevent
+#  event_extra       - extra functions, contains http, dns ans rpc
+#  event             - sum of two above, deprecated
+#  event_pthreads    - multiple threads for libevent, not exists on Windows
+#  event_openssl     - openssl support for libevent
```

Comment:
Hm, don't you think stripping `event_` from the name would be better?

Reply:
Like `boost`

Reply:
The new usage:
```
set(LIBEVENT_STATIC_LINK TRUE)
find_package(libevent 2.2 REQUIRED COMPONENTS core extra openssl pthreads)
target_link_libraries(myapp ${LIBEVENT_LIBRARIES})
```

Reply:
After stripping `event_ `, the variables `LIBEVENT_STATIC_LIBRARIES` and `LIBEVENT_SHARED_LIBRARIES` are not functional now.